### PR TITLE
Track api

### PR DESCRIPTION
**Pull request by sukaant**

### CODE REVIEW

File: apps/discovery_api/lib/discovery_api_web/router.ex [hunk 1]
```diff
@@ -36,20 +36,24 @@ defmodule DiscoveryApiWeb.Router do
     plug(DiscoveryApiWeb.Plugs.NoStore)
   end
 
+  pipeline :telemetry do
+    plug Plug.Telemetry, event_prefix: [:phoenix, :endpoint]
+  end
+
   scope "/api/v2", DiscoveryApiWeb do
-    pipe_through([:reject_cookies_from_ajax, :verify_token, :add_user_details, :global_headers])
+    pipe_through([:reject_cookies_from_ajax, :verify_token, :add_user_details, :global_headers, :telemetry])
```
Comment (LtChae) [inline apps/discovery_api/lib/discovery_api_web/router.ex:44]: See if you can add this to a pipeline that everything passes through

File: apps/andi/mix.exs [hunk 2]
```diff
@@ -4,7 +4,7 @@ defmodule Andi.MixProject do
   def project do
     [
       app: :andi,
-      version: "0.42.1",
+      version: "0.42.3",
```
Comment (cameronmarsh) [inline apps/andi/mix.exs:7]: nbd, but you bumped this two versions

File: apps/telemetry_event/lib/telemetry_event/helper/add_init_server.ex [hunk 3]
```diff
@@ -0,0 +1,18 @@
+defmodule TelemetryEvent.Helper.AddInitServer do
+  @moduledoc false
+  alias TelemetryEvent.Helper.TelemetryEventHelper
+
+  def add_metrics_prometheus(child, app_name) do
```
Comment (LtChae) [inline apps/telemetry_event/lib/telemetry_event/helper/add_init_server.ex:5]: This probably shouldn't be part of the `AddInitServer` module

File: apps/andi/mix.exs (hunk 2 above)
Comment (sukaant) [reply apps/andi/mix.exs:7]: Just noticed, Thanks!

File: apps/telemetry_event/lib/telemetry_event/helper/add_metrics.ex [hunk 4]
```diff
@@ -2,14 +2,41 @@ defmodule TelemetryEvent.Helper.AddMetrics do
...
+        [
+          [
+            metric_name: "phoenix.endpoint.stop.duration",
+            tags: [:request_path, :method],
+            tag_values: fn %{conn: conn} ->
+              %{request_path: Map.get(conn, :request_path), method: Map.get(conn, :method)}
+            end,
+            metric_type: :distribution,
+            unit: {:native, :millisecond},
+            reporter_options: [buckets: [0.01, 0.025, 0.05, 0.1, 0.2, 0.5, 1]]
```
Comment (LtChae) [inline apps/telemetry_event/lib/telemetry_event/helper/add_metrics.ex:33]: New metric type here. This should make a histogram of some sort I think? Make sure it's outputting in a way we can consume in dev.

**Review by LtChae — APPROVED** [review]

**Pull request merged**